compiler-core/src/analyse.rs

```diff
@@ -1362,7 +1362,7 @@ fn get_type_dependencies(typ: &TypeAst) -> Vec<EcoString> {
             ..
         }) => {
             deps.push(match module {
-                Some(module) => format!("{}.{}", name, module).into(),
+                Some(module) => eco_format!("{}.{}", name, module),
```

Comment:
Oh wow I had no idea this was a thing!

Reply:
I discovered it also by looking at `ecow` crate. Definitely useful and avoid to convert it back to `EcoString`. It can probably be used everywhere in the codebase 🙂